.eslintrc.js

```diff
@@ -2,19 +2,19 @@ module.exports = {
     extends: "airbnb",
     plugins: ["react"],
     env: {
-      "browser": true,
+      browser: true,
     },
     parser: "babel-eslint",
     rules: {
       "no-tabs": "off",
       "react/jsx-uses-react": "error",
       "react/jsx-uses-vars": "error",
       "comma-dangle": ["error", {
-        "arrays": "always-multiline",
-        "objects": "always",
-        "imports": "never",
-        "exports": "never",
-        "functions": "always-multiline"
+        arrays: "always-multiline",
+        objects: "always",
+        imports: "always",
+        exports: "always",
+        functions: "always-multiline"
```

Comment:
Trailing comma? If your conf is JS, you should lint it too :)

Reply:
done